scripts/cluster/common/utils.py

```diff
@@ -20,7 +20,7 @@ def try_set_file_permissions(file):
     os.chmod(file, 0o660)
     try:
         shutil.chown(file, group='microk8s')
-    except:
+    except LookupError:
```

Comment:
Why is this a LookupError? https://docs.python.org/3/library/shutil.html#shutil.chown

Reply:
The group not existing will trigger a `LookupError` (based on testing this myself). The file not existing will create a `FileNotFoundError`. That should never happen though, since this function is only used in code that creates the file first. It would be useful to have `FileNotFoundError` then crash the program, since that would be an actual bug.